### ISSUE TITLE
Not getting notifications of Pokemon

### ISSUE THREAD

**Issue by claimer21** [seikur0/PGO-mapscan-opt#248]
I'm running the scanner and one problem I keep having is that a rare Pokemon will not send me a pushbullet notification after the scanner has been running for over an hour or so.  I can still see that pokemon on the map, but I have to restart the scanner for the notifications to come through, which is inconvenient if I am out of the house.

There are no errors present in the python script, so I'm not sure what's going on.  I am running a webhook server for only twitter updates, and using the native pushbullet integration as well (without webhook server).

**Comment by seikur0** [comment]
I'm not sure how to help you, I put in an additional error message for pushbullet "connection aborted" errors and let it retry for these, maybe that's it. I can't help you on the PokeAlarm side though.

**Comment by claimer21** [comment]
Yeah, that's what was throwing me off was that the pushbullet stopped as well as the PokeAlarm.  We'll see if that helps.  Thanks as always, you're really quick man ;)

**Comment by seikur0** [comment]
I'm trying my best haha At that moment I was already working on it.

One other thing, that comes to mind, maybe you got a notification like half an hour earlier or something before together with the message, that it'll be hidden for 15 min and then back for 15 min? Because Pushbullet is really independent, I don't even prefilter the webhooks, that go to it, and it's weird that both would stop working.

**Comment by ryugamimanta** [comment]
I've added pushbullet notifications for my map a couple of days ago and set it up to push to a channel so a couple of others could get the pokemon notifications as well, but channel pushes stopped working for me today. Is it a pushbullet issue? I went to the actual channel webpage which is https://www.pushbullet.com/channel?tag=darklordmanta and the script pushes show up, but on my pushbullet windows app OR android app, my notifications are blank (i even deleted the old channel and made this new one, that's why there's almost no notifications on it). If i set it up to not push to channel, just to my account, my notifications work fine, but i'm the only one who can get them. It'd be nice if channel pushes worked again. Just wondering if my issue is similar or related to yours.

**Comment by seikur0** [comment]
It might be worth looking into telegram for both of you. It's easy to set up and free.

**Comment by acesplit** [comment]
I am having the same issue. I have been pushing to a channel for several weeks and as of today, it stopped working.
